app/assets/javascripts/discourse/app/templates/components/sidebar/section-link.hbs

```diff
@@ -7,14 +7,39 @@
     @current-when={{@currentWhen}}
     @title={{@title}}
   >
+    {{#if @prefixValue }}
+      <span class="sidebar-section-link-prefix {{@prefixType}} {{@prefixCSSClass}}" style={{this.prefixCSS}}>
```

Comment:
I wonder if we need a `@prefixCSSClass` here. Will just providing `sidebar-section-link-prefix-@prefixType` be sufficient?

Reply:
No, I am using it to mark chat avatar with green ring when user is active:

Code from chat plugin
```
            get prefixCSSClass() {
              const activeUsers = this.chatService.presenceChannel.users;
              const user = this.channel.chatable.users[0];
              if (
                !!activeUsers?.findBy("id", user?.id) ||
                !!activeUsers?.findBy("username", user?.username)
              ) {
                return "active";
              }
              return "";
            }
```

Reply:
ooo icic. This is fine then.